src/core/Table/Table.test.tsx

```diff
@@ -1044,23 +1044,37 @@ it('should pass custom props to row', () => {
   expect(element).toBeTruthy();
 });
 
-it('should render condensed table', () => {
-  const { container } = renderComponent({
-    density: 'condensed',
-  });
-
-  const table = container.querySelector(
-    '.iui-table.iui-condensed',
-  ) as HTMLElement;
-  expect(table).toBeTruthy();
-});
-
-it('should render extra condensed table', () => {
-  const { container } = renderComponent({
-    density: 'extra-condensed',
-  });
-  const table = container.querySelector(
-    '.iui-table.iui-extra-condensed',
-  ) as HTMLElement;
-  expect(table).toBeTruthy();
-});
+it.each(['condensed', 'extra-condensed'] as const)(
+  'should render condensed and extra condensed tables',
+  (density) => {
+    const { container } = render(
```

Comment:
Couldn't you write the same as you had written before? 
![image](https://user-images.githubusercontent.com/36186912/129589294-bdc861c0-6752-400e-8a59-44739b93276d.png)

Reply:
Oh yes, my bad I was trying to follow the other example

Reply:
Can you also change line 1048 to use `%s` instead of `condensed and extra condensed`?